src/test/coverartutils_test.cpp

```diff
@@ -34,13 +34,15 @@ class CoverArtUtilTest : public MixxxTest, public CoverArtCache {
 
 const QString kCoverLocationTest("res/images/library/cover_default.png");
 const QString kTrackLocationTest(QDir::currentPath() %
-                                 "/src/test/id3-test-data/cover-test.mp3");
+                                 "/src/test/id3-test-data/cover-test-png.mp3");
 
 TEST_F(CoverArtUtilTest, extractEmbeddedCover) {
     const QString kTestPath(QDir::currentPath() % "/src/test/id3-test-data/");
     QImage cover;
     // We never need to acquire security tokens for tests since we don't run
     // them in a sandboxed environment.
+	
+    // TODO(XXX) we need to compare with a reference file to ensure that we are reading correct data
```

Comment:
You would just need to load the reference image and compare against that instead of just checking if we did load any image at all,